config/vendor-api-v1.yml

```diff
@@ -591,8 +592,22 @@ components:
         uk_residency_status:
           type: string
           maxLength: 256
-          description: The candidate’s UK residency status indicating their right to work and study in the UK
+          description: The candidate’s UK residency status indicating their right to work and study in the UK. The candidate can provide details as free text for example "Settled status".
           example: UK Citizen
+          enum:
+          - UK Citizen
+          - Irish Citizen
+          - right_to_work_or_study_details
```

Comment:
I am leaning towards not amending the current `uk_residency_status` with this new enum, to avoid any confusion mainly as the other examples seem to be free text while this one is more like a variable